src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -90,6 +91,7 @@
         oneLineSummary = "Call somatic SNPs and indels via local re-assembly of haplotypes",
         programGroup = VariantProgramGroup.class
 )
+@DocumentedFeature
```

Comment:
Does the comms team want to reserve this annotation for tools whose docs are complete and up-to-date?

Even if so, it's good to have some output initially for the gatkDoc task in protected.